src/Orleans.Providers.RavenDB/Orleans.Providers.RavenDB.csproj

```diff
@@ -5,12 +5,14 @@
 		<RootNamespace>Orleans.Providers.RavenDb</RootNamespace>
 		<ImplicitUsings>enable</ImplicitUsings>
 		<Nullable>enable</Nullable>
+		<TreatWarningsAsErrors>false</TreatWarningsAsErrors>
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.Orleans.Reminders" Version="9.1.2" />
-		<PackageReference Include="Microsoft.Orleans.Server" Version="9.1.2" />
-		<PackageReference Include="RavenDB.Client" Version="6.2.0" />
+		<PackageReference Include="Microsoft.Orleans.Reminders" Version="9.2.0-preview3" />
```

Comment:
why are you bumping the dependencies to preview version? it should stay as is?

Reply:
done